source/winAPI/winUser/__init__.py

```diff
@@ -1,4 +1,14 @@
 # A part of NonVisual Desktop Access (NVDA)
-# Copyright (C) 2022 NV Access Limited
+# Copyright (C) 2022 NV Access Limited, Cyrille Bougot
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
+

```

Comment:
we may want to consider keeping `__init__` files empty(-ish).
I imagine we will be creating many `user32` function wrappers, which may clutter whatever else we may need to add to this file.

Perhaps we should have a `wrappers.py` or `functions.py`?

Reply:
File structure modified as required in c118f3be335df4638ff303edaebd9851bc17ba6c.